content/geoip/release-notes.mdx

```diff
@@ -7,6 +7,22 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2021-08-31" title="Mapping more mobile networks to a state">
+We will be mapping more mobile (cellular) networks to a state (subdivision)
+rather than a city for the United States and other countries. This change will
+better reflect the end user location of these IP addresses across a region
+instead of a specific city.
+
+For example, many of these networks use IP addresses across a region like
+“New England.” In such cases, we will map IP addresses to a state in the
```

Comment:
how about "may map" instead of "will map" - as it is still a work in progress, and sometimes we still map to city level when it should be state level.

Reply:
should be updated in the commit below